packages/datagrid/src/components/DatasetSerializer/datasetSerializer.test.js

```diff
@@ -209,6 +209,12 @@ describe('#getColumnDefs', () => {
 		expect(columnDefs).toEqual([]);
 	});
 
+	it('should remove the sub type', () => {
+		const columnDefs = getColumnDefs(sample, { hideSubType: true });
+
+		expect(columnDefs).toMatchSnapshot();
```

Comment:
Please stop snapshooting everything, write proper asserts to check exactly what you want
- snapshots are hard to read
- it's harder to see what is really tested
- changes in other part of the snapshot will make this test fails

Snapshots are ok to check markups but for simple tests, make simple assertions.

Reply:
ok I change it